### PR TITLE
update release ci

### LINKED ISSUE #24: add CI for release
pre-req #4 #23

### PR DESCRIPTION
**Pull request by sozercan**
Signed-off-by: Sertac Ozercan <sozercan@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #24 

**Special notes for your reviewer**:

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -42,7 +35,7 @@ jobs:
           path: |
             ~/go/pkg/mod
             ~/.cache/go-build
-      - uses: crazy-max/ghaction-github-runtime@v1
```
Comment (cpuguy83) [inline .github/workflows/release.yaml:45]: How come this is removed?

Comment (cpuguy83) [reply .github/workflows/release.yaml:45]: This lets us publish the build cache from buildkit to gha (unless something has changed in the last several months).

Comment (sozercan) [reply .github/workflows/release.yaml:45]: thanks, reverted

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Pull request merged**